### PR TITLE
Add separation dependency of files

### PR DESCRIPTION
**Pull request by igorkruglyak**

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pre-commit==1.17.0
+pylint==2.3.1
+pylint-django==2.0.11
```
Comment (webknjaz) [inline test-requirements.txt]: This line is unrelated to this PR

**Comment by webknjaz** [comment]
> Add separation dependency of files

"Put test deps into requirements file"

File: test-requirements.txt (hunk 1 above)
Comment (igorkruglyak) [reply test-requirements.txt]: Ok, how do I fix it?

Comment (webknjaz) [reply test-requirements.txt]: Easy: don't add this line.

Comment (igorkruglyak) [reply test-requirements.txt]: What is wrong now?

**Comment by webknjaz** [comment]
@igorkruglyak why did you delete all of the dependencies? You're supposed to just move them into a file.

File: test-requirements.txt (hunk 1 above)
Comment (webknjaz) [reply test-requirements.txt]: You erased two lines more than needed.

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
```
Comment (webknjaz) [inline test-requirements.txt]: This line is still not needed.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Review by webknjaz — APPROVED** [review]

**Pull request merged**